project/Dependencies.scala

```diff
@@ -25,19 +27,21 @@ object Dependencies {
     "com.getsentry.raven"            % "raven-logback"       % "7.8.6",
     "com.typesafe.scala-logging"    %% "scala-logging"       % "3.7.2",
 
-    "org.broadinstitute.dsde.vault" %% "vault-common"        % "0.1-19-ca8b927"
-      exclude("io.spray", "spray-routing_2.11"),
+    "org.parboiled" % "parboiled-core" % "1.2.0",
```

Comment:
this used to be a part of `vault-common`

Reply:
ah, I love that we no longer depend on `vault-common`